uploader/utils.py

```diff
@@ -17,9 +17,9 @@
 logging.basicConfig(stream=sys.stdout, level=logging.DEBUG)
 logger = logging.getLogger(__name__)
 
-PRODUCT_PATTERN = ".*-\\d+[.]\\d+[.]\\d+-ubuntu(0|[1-9][0-9]*)-(20\\d{2})[01][0-9][0-3][0-9][0-2]\\d[0-5]\\d[0-5]\\d\\S*"
+PRODUCT_PATTERN = ".*-\\d+[.]\\d+[.]\\d+.*-ubuntu(0|[1-9][0-9]*)-(20\\d{2})[01][0-9][0-3][0-9][0-2]\\d[0-5]\\d[0-5]\\d\\S*"
```

Comment:
thought: isn't `.*` regex too permissive? If all we want to add is a `preview1`, maybe just add `preview\d` as optional?